entries/s/schibsted.com.json

```diff
@@ -2,37 +2,35 @@
   "Schibsted": {
     "domain": "schibsted.com",
     "additional-domains": [
-      "schibsted.no",
-      "schibsted.fi",
-      "schibsted.dk",
-      "vg.no",
-      "finn.no",
-      "blocket.se",
-      "e24.no",
-      "aftonbladet.se",
-      "dba.dk",
       "adressa.no",
-      "tronderbladet.no",
+      "aftenbladet.no",
+      "aftenposten.no",
+      "aftonbladet.se",
       "bt.no",
+      "e24.no",
       "fvn.no",
-      "svd.se",
-      "tv.nu",
-      "aftenbladet.no",
-      "klart.se",
-      "omni.se",
-      "bytbil.com",
       "godt.no",
+      "klart.se",
       "minmote.no",
+      "omni.se",
       "pent.no",
+      "podme.com",
+      "schibsted.dk",
+      "schibsted.fi",
+      "schibsted.no",
+      "schibsted.se",
+      "svd.se",
       "tek.no",
-      "oikotie.fi",
-      "tori.fi"
+      "tronderbladet.no",
+      "tv.nu",
+      "vg.no"
     ],
     "tfa": [
       "sms",
       "totp"
     ],
-    "documentation": "https://login.schibsted.com/profile-pages/faq#how-to-activate-and-deactivate-2fa",
```

Comment:
To me, this URL points to a region-specific documentation page, whilst the documentation page you provided only provides the information in Norwegian.

Reply:
https://login.schibsted.com/profile-pages/faq#how-to-activate-and-deactivate-2fa Is in Swedish while https://payment.schibsted.no/profile-pages/faq#how-to-activate-and-deactivate-2fa is in Norwegian 

They both have language selection at the bottom 
![IMG_2410](https://github.com/user-attachments/assets/0521b66b-1020-41cd-a9dd-39f6882b72eb)

so since 2facorauth does not support localised links I updated it to Norway where the HQ and CEO reside 
Feel free to come with alternative ideas ☺️